src/components/ActionsHistory/ActionItem.tsx

```diff
@@ -17,7 +18,12 @@ export const ActionItem: React.FC<IAction> = (props) => {
   let address = props.address === props.recipient ? "From:  " : "To:  ";
   address += ellipsis(`${props.recipient || ""}`);
   return (
-    <Container>
+    <Container
+      onClick={() =>
+        props.actionHash &&
+        window.open(`https://iotexscan.io/action/${props.actionHash}`, "_blank")
```

Comment:
put to the wallet core layer, and also we should support testnet.iotexscan.io

in the future we may have other kinds of tokens from other sources.